src/app/book-list/book-list.component.html

```diff
@@ -2,20 +2,15 @@ <h2 id="title"> Books </h2>
   <div class="container">
     <div class="row">
       <div class="col-md-3" *ngFor="let book of books" (click)="onSelect(book)">
-          <img class="img-responsive" *ngIf="book.PictureURL" src="{{book.PictureURL}}" alt="Generic placeholder image">
-          <img class="img-responsive" *ngIf="!book.PictureURL" src="https://www.anglo-egyptian.com/books_posters/defbookcover.jpg" alt="Generic placeholder image">
-          <!--<div class="media" *ngIf="selectedBook===book&&book">-->
-              <!--<span class="book-name">{{book.Name}}</span>-->
-              <!--&lt;!&ndash;<img class="d-flex align-self-start mr-3" *ngIf="book.PictureURL" src="{{book.PictureURL}}" alt="Generic placeholder image">-->
-              <!--<img class="d-flex align-self-start mr-3" *ngIf="!book.PictureURL" src="https://www.anglo-egyptian.com/books_posters/defbookcover.jpg" alt="Generic placeholder image">&ndash;&gt;-->
-              <!--<div class="media-body">-->
-                <!--<h5 class="mt-0">Description:</h5>-->
-              <!--</div>-->
-              <!--<div class="d-block">-->
-                <!--<button class="btn btn-info btn-xs" (click)="goToDetail(selectedBook.Id)">Update</button>-->
-                <!--<button id="delete" class="btn btn-danger btn-xs" (click)="delete(book.Id); $event.stopPropagation()">Delete</button>-->
-              <!--</div>-->
-          <!--</div>-->
+          <div class="card">
+            <img class="card-img-top" *ngIf="book.PictureURL" src="{{book.PictureURL}}" alt="Card image cap">
+            <img class="card-img-top" *ngIf="!book.PictureURL" src="https://www.anglo-egyptian.com/books_posters/defbookcover.jpg" alt="Card image cap">
+            <div class="card-block">
+              <h4 class="card-title">{{book.Name}}</h4>
+              <p class="card-text">by {{book.Author}}</p>
+              <button type="button" class="btn btn-secondary btn-large" (click)="goToDetail(book.Id)"> View Details </button>
+            </div>
```

Comment:
There are quite many books. Perhaps using pagination would provide a better UX and better performance?

https://ng-bootstrap.github.io/#/components/pagination/api